lib/apis/v3/node.go

```diff
@@ -67,6 +67,9 @@ type NodeBGPSpec struct {
 	IPv6Address string `json:"ipv6Address,omitempty" validate:"omitempty,cidrv6"`
 	// IPv4IPIPTunnelAddr is the IPv4 address of the IP in IP tunnel.
 	IPv4IPIPTunnelAddr string `json:"ipv4IPIPTunnelAddr,omitempty" validate:"omitempty,ipv4"`
+	// RouteReflectorClusterID enables this node as a route reflector, automatically peering with
+	// other nodes in the same cluster.
+	RouteReflectorClusterID string `json:"routeReflectorClusterID,omitempty" validate:"omitempty"`
```

Comment:
Yea, that comment is wrong (from an earlier version of this experiment).

I do expect that it will work for a calico/node instance to perform local programming and act as a RR. Seems to work from my playing around.